shared-bindings/displayio/ColorConverter.c

```diff
@@ -110,9 +110,41 @@ const mp_obj_property_t displayio_colorconverter_dither_obj = {
               (mp_obj_t)&mp_const_none_obj},
 };
 
+//|     def make_transparent(self, pixel: int) -> None:
+//|         """Sets a pixel to not opaque."""
+//|
+STATIC mp_obj_t displayio_colorconverter_make_transparent(mp_obj_t self_in, mp_obj_t transparent_color_obj) {
+    displayio_colorconverter_t *self = MP_OBJ_TO_PTR(self_in);
+
+    mp_int_t transparent_color;
+    if (!mp_obj_get_int_maybe(transparent_color_obj, &transparent_color)) {
```

Comment:
Use the non-`_maybe` version here. That way it'll raise a standard error. Custom error strings are nice but they take up space.